app/src/Views/HomeView.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import IntroSection from "../Components/Home/IntroSection";
+import FeaturesPitch from "../Components/Home/FeaturesPitch";
+import AnalyticsPitch from "../Components/Home/AnalyticsPitch";
+import PaymentPitch from "../Components/Home/PaymentPitch";

```

Comment:
```suggestion
import PaymentPitch from "../components/Home/PaymentPitch";
```
* since, the folder name for components are `components`
* make this change for imports with `components`